.github/ISSUE_TEMPLATE/ci-sev.md

```diff
@@ -1,12 +1,10 @@
 ---
 name: "⚠️ CI SEV"
 about: Tracking incidents for PyTorch's CI infra.
+label: ["ci: sev"]
```

Comment:
lol wait, I was bitten by this seemingly convenient step a while ago.  Basically, random people will create issue using this template, and with the label is added automatically, the spam will show up everywhere.  We explicitly ask oncall to manually add this label to avoid that because only people with write permission to the repo can do so